.github/workflows/workflow.yml

```diff
@@ -197,3 +197,31 @@ jobs:
           echo "${{ secrets.FRANCE_API_DEPLOY_KEY }}" > /home/runner/.ssh/france_api_deploy
           chmod 600 /home/runner/.ssh/france_api_deploy
           ssh -i /home/runner/.ssh/france_api_deploy -o StrictHostKeyChecking=no deploy-api@fr.openfisca.org
+
+  check-api-version:
+    runs-on: ubuntu-latest
+    needs: [ deploy ]
+    outputs:
+      version-match: ${{ steps.check-api-version.outputs.version-match }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Cache build
+        id: restore-build
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-build-${{ hashFiles('setup.py') }}
+      - name: Check api version
+        id: check-api-version
+        run: |
+          output=$(python "${GITHUB_WORKSPACE}/.github/check-api-version.py")
+          echo "::set-output name=version-match::$output"
```

Comment:
Relying on string outputs and comparisons seems a bit complex.
Couldn't we instead use on standard UNIX exit codes? 🙂
The `check-api-version.py` script could exit with `0` if the versions match, and with `1` if they don't, printing both versions to `stdout` in any case and a message to `stderr` if they don't.